### ISSUE TITLE
what "segment-based trajectory contains a series of GPS locations" means?

### ISSUE THREAD

**Issue by xpf2333** [vonfeng/DeepMapMatching#2]
In the section of problem defining, you say " In the road network, each road is split into a series of segments and each segments contains a series of GPS locations. ". The segments are based OSM, what is the relationship between segments and GPS location? Thank you!

**Comment by xpf2333** [comment]
After reading your code, I find that the correspondence between seg_id and GPS locations appears to be known first. Is that correct? Thank you!

**Comment by vonfeng** [comment]
Yes, the relation between road segment and GPS are known during the experiment.

**Comment by xpf2333** [comment]
> Yes, the relation between road segment and GPS are known during the experiment.

感谢feng老师的回复！我想再问下您的意思是，数据集中的某个点A，实际上已经被对应到了某个segment上的点B吗？这个数据集仅是用于训练模型的。那么当用模型进行推断时，一个原始序列被输入时，该如何进行calibration的过程呢。谢谢！
